### ISSUE TITLE
[Bug][Series][Label] Series data point labels are not visible when chart is saved as image file

### ISSUE THREAD

**Issue by YogeshK-Itanta** [apache/echarts#19539]
### Version

5.3.2

### Link to Minimal Reproduction

https://echarts.apache.org/examples/en/editor.html?c=line-simple&code=PYBwLglsB2AEC8sDeAoWsBEAjAhgYwGsBzAJ2AFdoATAYWABtgSMAuTDAGjUxIFMBnEDH4QAbr1awwJcry7oMYYA0ghJqdAukQiRXszYYcADwj9O3BfwAWwAO6Tps-ZsW9jYAMpgAnvQlsGpoKAGYwXhAAXgGwAIwATJawAL7cyS6KyvRYwMbqSRgkOtZgkrEAbC6hvDhg5Hz5wQoAtjgkBI1NVrYObE68Sejpg5itRBB4ACo-IDFBXRg29pIhOPT8A13DC3z8Sg2BI93LfTKbTanBlykZVLU4AFrKzZ1aMzEYENAiVBJpGaQIFRXpg8OEcF8ADI4LC8eiOM7_bgYURmchrACyODUbAA2gBdDL-PTUEGKd6SRZ4Mj0eFVTBrHTQSn-EKlemfMC8ZoACV4xVKbFiAAYORAuc0AOpAsDWMqigpcjzePxzEYYMLQCLReVJa7bTDGACCpnMh2C5NmlMgzQkHLuYBwkgJHOgOFtlIsFrdtshwDwtSgzMMzSBVH8XtcPt4k3cXl8EfNTQ14U8URisQALPTQuFJfyiCVKTl6L9mDnMOISJAA_QjfQmdbQJGLSAcFQqF8iM6jrAAMyVXsKrroYcj4UjfF613u3gAcWxkjHCkZRGgkwmBDNUjOHJMZmhsPhSYtYMYBh4RFwAApRbA73fYgBKFuuMiOrmeisYYBVkKMXpMCcaAA0_b9rCBXgAHkq3oRc2FWdY5HVJY7CxYxDzhBFnBQnoMShGEsNOHCFk1bUM2zdUwhIVowC5C8MCQHxmJ8ZIAFokAxDF2KQDtkgAHWgJAeR5ZIWCQZpmjEpB-H4ZIMGnAp934DdCBBY5AP6b9V2gaVZUw48d1kRSLRySg7hIHwFxxIzkItQQGzAAA5chmlhC9yiRBQfBNMwe2CeZXF8K1DFENZZFfBRoy_dUdNUrdsLs5MhBESAYBZXg2UihlTQM9TXDPJhKQAYkzAARcqADFKuyhQIN-GD9DgmzEI2CsNPQvLiKShZUPw6Auts9rMDItMdSFRItm03KvjVEdMFQxLhowVoSHGYN72WhtoF4FVE2QXsFEKhjSoqsrqoU-aDQWGAHn0YAVjWDYRmu1wHPFFy3P0GKFmjP1QKDSleFJb9o1jZUEzmkcUy1MaM3iZayPzAViwYMtLouPVuEJZENiKAR_M0QKFAdJ08V7XFDricoAFYAHZ4hpgAOeJhTZtnhvQGnygAOkmkccfmyn5vQCp6ficoAE52fZznYCZvseb7XtBZHYWRbFhm6clpmZeXEdJczPmVeG9X5s1-JpfKPW5cl2Ieclk2KapzW-2FLMbap3Wec8-bVa6M2R1dhJWZluWFZ5pnHb902Xdpum-3iN3PZF2AE4dp2hbj-m-1z3Ww6pummZ52JM7V7OE8zTNrYL1O-3t0u_cnb9VoII0-DJ5BXreELMG2u1cPsTwfDchglpQkeSzhpdvzgo98o08f5owFLxUBwwlDUZbjuLOC1MR1N0zKfmrm_JQVAgGykG7hbmmUWVHqQ0HZ0pAAFexvu_MFoB2vBnPIWk24tIoUdIQbwJBai8CID4Skaw6QgPwB0Qw2VsBtHQtKKgD82DW2_OKbke0oYLB3oYU6VUaovSxugKcyQADcQA

### Steps to Reproduce

- Plot a chart. (options can be found in the minimal reproduction link)
- Make sure that labels for series data points are visible.
- Create an image with the same chart as:
``` 
  const data = canvas.toBuffer('image/png');
  fs.writeFileSync(filename, data, { mode: 0o644, flag: 'w' });
```
- I do not see labels on data points of the series.

### Current Behavior

- I create my chart with:

``` 
  const canvas = createCanvas(width, height);
  const chart = echarts.init(canvas);
  chart.setOption(options);
  //options can be found in the minimal reproduction link
```

- This renders the chart as expected. (Demo can be found [here](https://echarts.apache.org/examples/en/editor.html?c=line-simple&code=PYBwLglsB2AEC8sDeAoWsBEAjAhgYwGsBzAJ2AFdoATAYWABtgSMAuTDAGjUxIFMBnEDH4QAbr1awwJcry7oMYYA0ghJqdAukQiRXszYYcADwj9O3BfwAWwAO6Tps-ZsW9jYAMpgAnvQlsGpoKAGYwXhAAXgGwAIwATJawAL7cyS6KyvRYwMbqSRgkOtZgkrEAbC6hvDhg5Hz5wQoAtjgkBI1NVrYObE68Sejpg5itRBB4ACo-IDFBXRg29pIhOPT8A13DC3z8Sg2BI93LfTKbTanBlykZVLU4AFrKzZ1aMzEYENAiVBJpGaQIFRXpg8OEcF8ADI4LC8eiOM7_bgYURmchrACyODUbAA2gBdDL-PTUEGKd6SRZ4Mj0eFVTBrHTQSn-EKlemfMC8ZoACV4xVKbFiAAYORAuc0AOpAsDWMqigpcjzePxzEYYMLQCLReVJa7bTDGACCpnMh2C5NmlMgzQkHLuYBwkgJHOgOFtlIsFrdtshwDwtSgzMMzSBVH8XtcPt4k3cXl8EfNTQ14U8URisQALPTQuFJfyiCVKTl6L9mDnMOISJAA_QjfQmdbQJGLSAcFQqF8iM6jrAAMyVXsKrroYcj4UjfF613u3gAcWxkjHCkZRGgkwmBDNUjOHJMZmhsPhSYtYMYBh4RFwAApRbA73fYgBKFuuMiOrmeisYYBVkKMXpMCcaAA0_b9rCBXgAHkq3oRc2FWdY5HVJY7CxYxDzhBFnBQnoMShGEsNOHCFk1bUM2zdUwhIVowC5C8MCQHxmJ8ZIAFokAxDF2KQDtkgAHWgJAeR5ZIWCQZpmjEpB-H4ZIMGnAp934DdCBBY5AP6b9V2gaVZUw48d1kRSLRySg7hIHwFxxIzkItQQGzAAA5chmlhC9yiRBQfBNMwe2CeZXF8K1DFENZZFfBRoy_dUdNUrdsLs5MhBESAYBZXg2UihlTQM9TXDPJhKQAYkzAARcqADFKuyhQIN-GD9DgmzEI2CsNPQvLiKShZUPw6Auts9rMDItMdSFRItm03KvjVEdMFQxLhowVoSHGYN72WhtoF4FVE2QXsFEKhjSoqsrqoU-aDQWGAHn0YAVjWDYRmu1wHPFFy3P0GKFmjP1QKDSleFJb9o1jZUEzmkcUy1MaM3iZayPzAViwYMtLouPVuEJZENiKAR_M0QKFAdJ08V7XFDricoAFYAHZ4hpgAOeJhTZtnhvQGnygAOkmkccfmyn5vQCp6ficoAE52fZznYCZvseb7XtBZHYWRbFhm6clpmZeXEdJczPmVeG9X5s1-JpfKPW5cl2Ieclk2KapzW-2FLMbap3Wec8-bVa6M2R1dhJWZluWFZ5pnHb902Xdpum-3iN3PZF2AE4dp2hbj-m-1z3Ww6pummZ52JM7V7OE8zTNrYL1O-3t0u_cnb9VoII0-DJ5BXreELMG2u1cPsTwfDchglpQkeSzhpdvzgo98o08f5owFLxUBwwlDUZbjuLOC1MR1N0zKfmrm_JQVAgGykG7hbmmUWVHqQ0HZ0pAAFexvu_MFoB2vBnPIWk24tIoUdIQbwJBai8CID4Skaw6QgPwB0Qw2VsBtHQtKKgD82DW2_OKbke0oYLB3oYU6VUaovSxugKcyQADcQA))

- But when I create an image with the same chart as:

``` 
  const data = canvas.toBuffer('image/png');
  fs.writeFileSync(filename, data, { mode: 0o644, flag: 'w' });
```
I do not see labels on data points of the series.
![echarts-1706075060739](https://github.com/apache/echarts/assets/127471176/6b390fc5-67d2-475b-969b-3a2785a1e366)

### Expected Behavior

Chart should save the image with labels like this:
<img width="719" alt="image" src="https://github.com/apache/echarts/assets/127471176/ec725abd-3893-4ce0-8b36-237119476eb7">

### Environment

```markdown
- OS: Windows 11 pro 22H2
- Node: 20.10.0
```

**Comment by MatthiasMert** [comment]
You can use the [saveAsImage](https://echarts.apache.org/en/option.html#toolbox.feature.saveAsImage) feature instead.

**Comment by Ovilia** [comment]
Yes, please use `saveAsImage`. Getting the `canvas` is not suggested because it breaks the capsulization and may break things down.
